A bunch of resize tests are failing on Python3.7. Any idea why?
```
E           ValueError: cannot resize an array that references or is referenced
E           by another array in this way.
E           Use the np.resize function or refcheck=False
```
Seems to have come in in #12121.

--- a/.travis.yml
+++ b/.travis.yml
@@ -40,13 +40,13 @@ env:
 matrix:
   include:
     # Do all python versions without environment variables set
-    - python: 3.5
     - stage: Initial tests
-      python: 3.6
+      python: 3.8
+
+    - python: 3.5
+    - python: 3.6
     - python: 3.7
-    - python: 3.8-dev
-    - python: 3.7
-      env: INSTALL_PICKLE5=1
+
     - python: 3.6
       dist: bionic
       env: USE_DEBUG=1
@@ -58,11 +58,14 @@ matrix:
             - python3-dbg
             - python3-dev
             - python3-setuptools
+
     - python: 3.7
       env: USE_WHEEL=1 RUN_FULL_TESTS=1 RUN_COVERAGE=1 INSTALL_PICKLE5=1
-    - python: 3.6
+
+    - python: 3.7
       env: USE_SDIST=1
-    - python: 3.6
+
+    - python: 3.7
       env:
        - PYTHONOPTIMIZE=2
        - BLAS=None
@@ -71,24 +74,30 @@ matrix:
        - NPY_BLAS_ORDER=mkl,blis,openblas,atlas,accelerate,blas
        - NPY_LAPACK_ORDER=MKL,OPENBLAS,ATLAS,ACCELERATE,LAPACK
        - USE_ASV=1
-    - python: 3.5
+
+    - python: 3.7
       env: NPY_RELAXED_STRIDES_CHECKING=0
-    - python: 3.6
+
+    - python: 3.7
       env: USE_WHEEL=1 NPY_RELAXED_STRIDES_DEBUG=1
-    - python: 3.6
+
+    - python: 3.7
       env: NUMPY_EXPERIMENTAL_ARRAY_FUNCTION=0
-    - python: 3.6
+
+    - python: 3.7
       env:
        - BLAS=None
        - LAPACK=None
        - ATLAS=None
-    - python: 3.6
+
+    - python: 3.7
       os: linux
       arch: ppc64le
       env:
        # use ppc64le OpenBLAS build, not system ATLAS
        - ATLAS=None
-    - python: 3.6
+
+    - python: 3.7
       os: linux
       arch: s390x
       env:

--- a/tools/travis-test.sh
+++ b/tools/travis-test.sh
@@ -66,8 +66,16 @@ setup_base()
 run_test()
 {
   $PIP install -r test_requirements.txt
+
   if [ -n "$USE_DEBUG" ]; then
     export PYTHONPATH=$PWD
+  fi
+
+  # pytest aborts when running --durations with python3.6-dbg, so only enable
+  # it for non-debug tests. That is a cPython bug fixed in later versions of
+  # python3.7 but python3.7-dbg is not currently available on travisCI.
+  if [ -z "$USE_DEBUG" ]; then
+    DURATIONS_FLAG="--durations 10"
   fi
 
   if [ -n "$RUN_COVERAGE" ]; then
@@ -82,17 +90,15 @@ run_test()
     "import os; import numpy; print(os.path.dirname(numpy.__file__))")
   export PYTHONWARNINGS=default
 
-  if [ -n "$PPC64_LE" ]; then
+  if [ -n "$CHECK_BLAS" ]; then
     $PYTHON ../tools/openblas_support.py --check_version $OpenBLAS_version
   fi
 
   if [ -n "$RUN_FULL_TESTS" ]; then
     export PYTHONWARNINGS="ignore::DeprecationWarning:virtualenv"
-    $PYTHON -b ../runtests.py -n -v --durations 10 --mode=full $COVERAGE_FLAG
+    $PYTHON -b ../runtests.py -n -v --mode=full $DURATIONS_FLAG $COVERAGE_FLAG
   else
-    # disable --durations temporarily, pytest currently aborts
-    # when that is used with python3.6-dbg
-    $PYTHON ../runtests.py -n -v  # --durations 10
+    $PYTHON ../runtests.py -n -v $DURATIONS_FLAG
   fi
 
   if [ -n "$RUN_COVERAGE" ]; then

